Convert `process-maintenance-backlog.py` to python3

dewei-zhu opened the pull request:
#### 7cc51d56da500928c4320f60589c3d5ad759653e
<pre>
Convert `process-maintenance-backlog.py` to python3
<a href="https://bugs.webkit.org/show_bug.cgi?id=268858">https://bugs.webkit.org/show_bug.cgi?id=268858</a>
<a href="https://rdar.apple.com/122416143">rdar://122416143</a>

Reviewed by Jonathan Bedard.

Convert `process-maintenance-backlog.py` to python3 syntax.

* Websites/perf.webkit.org/tools/process-maintenance-backlog.py:
(main):
(ask_yes_no_question):
(submit_report):

Canonical link: <a href="https://commits.webkit.org/274199@main">https://commits.webkit.org/274199@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/72c0a7699ffb728586becd7c3f497c43fac13e9a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/38223 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/17165 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/40540 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/40780 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/34005 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/40457 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/19917 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/14507 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/32269 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/38796 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/14480 "Passed tests") | [⏳ 🧪 api-mac](https://ews-build.webkit.org/#/builders/API-Tests-macOS-EWS "Waiting to run tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/12620 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/12583 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/34184 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/42057 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/34734 "Passed tests") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/34617 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/38440 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/13175 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/10864 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/36639 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/14727 "Built successfully") | | | 
| [  ~~🛠 🧪 merge~~](https://ews-build.webkit.org/#/builders/19/builds/8606 "The change is no longer eligible for processing.") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/13591 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/4977 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/14191 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

xw commented on Websites/perf.webkit.org/tools/process-maintenance-backlog.py:
```diff
@@ -14,35 +14,36 @@ def main():
     server_config = load_server_config(args.server_config_json)
 
-    print 'Submitting results in "%s" to "%s"' % (maintenace_dir, server_config['server']['url'])
+    print(f'Submitting results in "{maintenance_dir}" to \"{server_config["server"]["url"]}\"')
```
Nit: One set of quotes is escaped and the other isn't. This would probably be more readable without escapes overall:
```
print(f'Submitting results in "{maintenance_dir}" to "{server_config["server"]["url"]}"')
```

dewei-zhu committed:
Convert `process-maintenance-backlog.py` to python3
https://bugs.webkit.org/show_bug.cgi?id=268858
rdar://122416143

Reviewed by Jonathan Bedard.

Convert `process-maintenance-backlog.py` to python3 syntax.

* Websites/perf.webkit.org/tools/process-maintenance-backlog.py:
(main):
(ask_yes_no_question):
(submit_report):

Canonical link: https://commits.webkit.org/274199@main
